selenium/chrome-debug/Dockerfile.txt

```diff
@@ -0,0 +1,45 @@
+#==============================
+# Scripts to run Selenium
+#==============================
+COPY entry_point.sh \
+  functions.sh \
+    /opt/bin/
+
+USER root
+
+#==============
+# Xvfb
+#==============
+RUN apt-get update -qqy \
+  && apt-get -qqy install \
+    xvfb \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+#=====
+# VNC
+#=====
+RUN apt-get update -qqy \
+  && apt-get -qqy install \
+  x11vnc \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+#=========
+# fluxbox
+# A fast, lightweight and responsive window manager
+#=========
+RUN apt-get update -qqy \
+  && apt-get -qqy install \
+    fluxbox \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+
+#============================
+# Some configuration options
+#============================
+ENV SCREEN_WIDTH 1360
+ENV SCREEN_HEIGHT 1020
+ENV SCREEN_DEPTH 24
+ENV DISPLAY :99.0
+
+EXPOSE 5900
+
+CMD ["/opt/bin/entry_point.sh"]
```

Comment:
did you mean to make this `entrypoint` ?

Reply:
Entrypoint is probably better. I'll update in a bit. I gotta bring the old compose file back in cnx-automation and see what happens.